collectors/pod_test.go

```diff
@@ -36,6 +38,9 @@ func TestPodCollector(t *testing.T) {
 	// Fixed metadata on type and help text. We prepend this to every expected
 	// output so we only have to modify a single place when doing adjustments.
 	var test = true
+	startTime := time.Now()
```

Comment:
Actually we're checking whether our expectation is our expectation here, could you do a constant time here and then write out the expected output?